example/spec/helpers/testUtils.js

```diff
@@ -89,6 +89,16 @@ function loadConfigFromKes(type) {
 const loadConfigFromYml = () => {
   const config = loadYmlFile('./config.yml');
 
+  // Pull stackname from env in Bamboo CI
+  if (!isNil(process.env.DEPLOYMENT) && isNil(config.stackName)) {
```

Comment:
Very nitpicky, but this would read a little better if the order of the two conditions were reversed.
```suggestion
  if (isNil(config.stackName) && !isNil(process.env.DEPLOYMENT)) {
```